cdk.tf/README.md

```diff
@@ -0,0 +1,3 @@
+# cdk.tf
```

Comment:
I wonder if we can make this live anywhere nested, like having a "websites" folder

Reply:
I'd refrain from having `website` and `websites` 😅 But I'm also not happy with it polluting the root of our repository 😁 However, it just didn't feel right in any of the other directories.
Ideally I'd like to put this into `website` but that directory already is a Vercel root 😅

Do you have any other idea where to put this? I mean, it could be buried as it is not that important to intuitively discover.

Reply:
Ah I see, I thought we could haven website, then this and the docs under it